src/main/java/ch/saunah/saunahbackend/service/BookingService.java

```diff
@@ -163,7 +176,7 @@ public Booking editBooking(int bookingId, BookingBody bookingBody) throws IOExce
         Objects.requireNonNull(bookingBody.getLocation(), "Location must not be null!");
         validateBookingData(bookingBody);
```

Comment:
does it now work to update? the last time I tested, it threw errors as it was not available (because of the booking itself)